src/HW3.java

```diff
@@ -0,0 +1,28 @@
+public class HW3 {
+    public static void main(String[] args) {
+        String string = "aaa bbbbb ccccccc";
+        String w = "";
+        String lw = "";
+
+        for (int i = 0; i < string.length(); i++) {
```

Comment:
it's better to place in a separate method. In the main method we just call methods - please do not write method implementation in the main method

Reply:
Good implementation example:
 static String wordGreatLength(String word1, String word2, String word3) {
        int wordlenght1 = word1.length();
        int wordlenght2 = word2.length();
        int wordlenght3 = word3.length();

        if ((wordlenght1 > wordlenght2) && (wordlenght1 > wordlenght3)) {
            return word1;
        } else if ((wordlenght2 > wordlenght1) && (wordlenght2 > wordlenght3)) {
            return word2;
        } else {
            return word3;
        }
    }